### PR TITLE
Support arm64 arch

### PR DESCRIPTION
**Pull request by jmontroy90**
Support ARM64 for architecture.

Fixes testing for the latest magefile version (v1.14.0) on ARM-based machines (most notably the new M1 / M2 Apple machines). Without this, tests were looking for a 32-bit Linux binary in tests, which is no longer produced by the upstream magefile repo, and wouldn't be right anyway.

Error output:
```
#13 6.453 Mage version found: v1.14.0
#13 6.454 Downloading https://github.com/magefile/mage/releases/download/v1.14.0/mage_1.14.0_Linux-32bit.tar.gz...
#13 6.454 ::debug::Downloading https://github.com/magefile/mage/releases/download/v1.14.0/mage_1.14.0_Linux-32bit.tar.gz
#13 6.454 ::debug::Destination /tmp/github_runner/b8051a76-30f0-4d00-b74c-6b9af3fcdb5e
#13 6.557 ::debug::Failed to download from "https://github.com/magefile/mage/releases/download/v1.14.0/mage_1.14.0_Linux-32bit.tar.gz". Code(404) Message(Not Found)
```

Now, fixed result is that `docker buildx bake test` passes successfully.

Note: I regenerated the `dist/` artifacts as needed too and committed those. I'm no TypeScript / Javascript dev, so hopefully that all looks okay.

### CODE REVIEW

File: src/installer.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export async function getMage(version: string): Promise<string> {
 
 const getFilename = (semver: string): string => {
   const platform: string = osPlat == 'win32' ? 'Windows' : osPlat == 'darwin' ? 'macOS' : 'Linux';
-  const arch: string = osArch == 'x64' ? '64bit' : '32bit';
+  const arch: string = osArch == 'x64' || osArch == 'arm64' ? '64bit' : '32bit';
```
Comment (crazy-max) [inline src/installer.ts:50]: Looking at [latest release](https://github.com/magefile/mage/releases/tag/v1.14.0), I think it should be:

```suggestion
  const arch: string = osArch == 'x64' || osArch == 'ARM64' ? '64bit' : '32bit';
```

Comment (jmontroy90) [reply src/installer.ts]: Ah, yea, thanks. Done.

Comment (jmontroy90) [reply src/installer.ts]: I guess there might need to be a handling of `arm32 -> ARM` as well, but I'm assuming that 32-bit ARM is what the `-ARM` suffix is about, and I also don't know that `uname -m` is `arm32` on those systems. Something to look into, or you happy with this as is?

**Review by crazy-max — APPROVED** [review]
LGTM thanks

**Pull request merged**